regulations/static/regulations/css/scss/partials/_sidebar_left.scss

```diff
@@ -29,11 +29,50 @@ aside.left-sidebar {
 
 		li {
 			list-style: none;
+			overflow: hidden;
+			max-height: 500em;
 
 			a {
 				text-decoration: none;
 				font-weight: bold;
 			}
+
+			&[data-state="collapsed"] {
+				max-height: 0;
+			}
+
+			transition: all .5s ease-out;
+		}
+
+		button {
+			border: none;
+			background: none;
+			color: $reverse_link_color;
+			cursor: pointer;
+			outline: none;
+
+			&:hover {
+				color: $reverse_link_hover;
+				@include transition;
+			}
+
+			&[data-state="expanded"] {
+				display: none;
+			}
+			&[data-state="expanded"] + button {
```

Comment:
I think I understand what's happening here but it's clearer to either give the buttons a class or select them with `data-set-state="collapsed"`

Reply:
Fixed in 98fa966